README.md

```diff
@@ -24,6 +24,7 @@ _Note: 1.28.0 and later require Gradle 7_
 * Update dependency versions
 * Add `substituteModuleDependecies` utility method for experimentation with some dependency substitution ([Issue 49316](https://www.labkey.org/home/Developer/issues/Secure/issues-details.view?issueId=49316))
 * Remove support for `skipBuild` property
+* Make `startTomcat` environment consistent between standalone and embedded Tomcat
```

Comment:
I'll plan to release 2.0.0 after this gets merged, so please update the date above before merging (just so I don't have to make another PR with that date update).